tests/unit/build_tests/BUILD

```diff
@@ -7,22 +7,23 @@ load("@bazel_skylib//rules:native_binary.bzl", "native_binary")
 build_test(
     name = "all_artifacts",
     targets = [
-        "@regression_testing_coursier//:org_pantsbuild_jarjar",
+        "@regression_testing_coursier//:build_buf_protovalidate",
```

Comment:
Lots of diff noise to sort the targets and add `"@regression_testing_coursier//:com_github_jnr_jffi_native"`